README.txt

```diff
@@ -1,5 +1,34 @@
 Generate executable via
 ```
-gcc -Wall -g -std=c11 -o tabReplace tabReplace.c
-gcc -Wall -g -std=c11 -o spaceReplace spaceReplace.c
+gcc -Wall -g -std=c11 -o tabReplace tabReplace.c fileProcessing.c replacement.c
+gcc -Wall -g -std=c11 -o spaceReplace spaceReplace.c fileProcessing.c replacement.c
```

Comment:
Possible future improvement, have a `generate.sh` script that runs the `gcc` commands for the user so they don't have to manually remember to add the included header files when compiling